kuronometer-core/src/main/scala/com/github/pedrovgs/kuronometer/free/domain/BuildExecution.scala

```diff
@@ -0,0 +1,37 @@
+package com.github.pedrovgs.kuronometer.free.domain
+
+object BuildTool extends Enumeration {
+  val Gradle, Maven, SBT = Value
+}
+
+object Platform extends Enumeration {
```

Comment:
Yeap, you are totally right. But for now we don't need more and I prefer to use this implementation instead of changing the json lib. In the future I'll use circe instead of lift json. Thanks for the tip!